common/forge-common.js

```diff
@@ -34,19 +34,23 @@
         // monitor #red-ui-header-button-sidemenu & add FlowForge links to the main menu
         monitorInsertion('#red-ui-header-button-sidemenu').summary(function (_arrayOfInsertedNodes) {
             if (!RED) { return }
+
+            // add Manage flowforge link
             const img = $('#red-ui-header > span > img')
-            const ownerHref = img.parent().prop('href')
-            if (ownerHref) {
+            const ownerHref = img.parent().prop('href') || 'https://app.flowforge.com'
+            try {
```

Comment:
Just belt and braces. 
If the user has entered a URL like `my.domain` it will throw an error (`new URL(...)` throws if there is no protocol/scheme or host/domain)

Reply:
The hard coding of `app.flowforge.com` is there to catch an issue where someone would be using the theme outside of flowforge on a stand alone node-red, or some possibility where the project url wasn't passed through from the config.

I'm not entirely sure why its reading it from the image I believe its something to do with setting the link on the image via CSS instead of JS but using the config param, Steve can explain better.

Reply:
The original code simply added a menu link that took the user back to the origin of the URL on the image that is applied by node-red reading its settings file.

e.g. 
| IMG URL | Menu Entry |
| - | - |
| _blank_ | _no_menu_entry_ |
| `https://localhost:3000/project/abcdef123456789` | `https://localhost:3000/` |
| `https://app.flowforge.com/project/defabc567891234` | `https://app.flowforge.com/` |



The original bug this PR intended to fix is **"Do NOT throw an error when there is no URL"**.

It was suggested that `app.flowforge.com` would be a good fallback as that would take users to our cloud offering - seemed a sensible enough fallback but I do realise that is not perfect as the user might just "like" the flowforge themes.

I am perfectly happy to return to NOT adding a "Manage FlowForge" menu item if the URL is not configured.

Please advise.

Reply:
OK, so rather than sit on this, I have decided the following direction...


The additional menu items will be added as requested by the [design Figma](https://www.figma.com/file/iEETlVUagOe9Lc2LseeMZH/FlowForge-Node-RED-Customisations?node-id=0%3A1)* only if the theme is running on NR under FlowForge (local/cloud/any).  This is determined by a regex test on the header image URL


![image](https://user-images.githubusercontent.com/44235289/178556652-647fe3d4-9be7-4750-9ffb-5b6c6aaa374d.png)

![image](https://user-images.githubusercontent.com/44235289/178556671-72c4959c-cf46-490b-8b5b-5a2f6ef378e2.png)

Otherwise, these 2 ↑ menu items will not be added

<br>

I will push this change & re-request review


---

`*` I realise the Figma was never ratified / bought off so many of the decisions were made to get things moving - so I may not have made all the right choices.  Perhaps a re-review of design is necessary now that we have "something"?